automation/check-merged.sh

```diff
@@ -2,6 +2,12 @@
 EXPORTED_DIR="$PWD/exported-artifacts"
 DOCS_DIR="$EXPORTED_DIR/docs"
 
+if hash dnf &>/dev/null; then
```

Comment:
Well, hash is a bash builtin while which is a separated command, hash will ignore aliases, and reads entries from the hash table and allows managing it, while which will not use hash tables but simulate bash behavior, and do the path searching each time.

In this case, you can just swap one with the other.